src/app.rb

```diff
@@ -0,0 +1,168 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    create_initial_data
+  end
+
+  # Create some initial data
+  def create_initial_data
+    # Create students
+    student1 = Student.new(15, name: 'John', parent_permission: true)
+    student2 = Student.new(16, name: 'Alice', parent_permission: true)
+
+    # Create a teacher
+    teacher1 = Teacher.new(35, name: 'Mr. Smith', specialization: 'Math')
+
+    # Create books
+    book1 = Book.new('Introduction to Ruby', 'John Rubyist')
+    book2 = Book.new('Ruby on Rails Guide', 'Ruby Rails')
+
+    # Create rentals
+    rental1 = Rental.new('2023-10-12', book1, student1)
+    rental2 = Rental.new('2023-10-10', book2, student2)
+
+    # Add the created objects to your data
+    @people.push(student1, student2, teacher1)
+    @books.push(book1, book2)
+    @rentals.push(rental1, rental2)
+  end
+
+  # List of books...
+  def list_books
+    puts(@books.map { |book| "Title: \"#{book.title}\", Author \"#{book.author}\"" })
+  end
```

Comment:
- [ ] When i click on option 1 to list all books at the start of the application, i can see that there are already created books as in the image below
![1](https://github.com/Kaiserabbas/school_library/assets/89055502/2c58a968-6aad-498f-adf6-bef258f8e268)
- [ ] After adding two books, i can also see the newly added books with the already added books as shown in the image below, however your app should not function this way. only books added at the start of the app should be dipslayed as done in this [reference video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
![2](https://github.com/Kaiserabbas/school_library/assets/89055502/2344a3ed-b6c9-4f2e-8f6c-bdce2c8d41fc)

Reply:
[Update the code by removing initial data supplied to the app](https://github.com/Kaiserabbas/school_library/pull/4/commits/5fb09cf98f3d48509812bb27f0ce5dbc12be1d40)